src/TileViewer.cpp

```diff
@@ -98,9 +94,6 @@ TileViewer::TileViewer(QWidget* parent)
             this, &TileViewer::imageMouseOver);
     connect(imageWidget, &VramTiledView::imageClicked,
             this, &TileViewer::displayCharInfo);
-
-	// and now go fetch the initial data
-	VDPDataStore::instance().refresh();
```

Comment:
Line 103 made redundant because of the connect in DebuggerForm.cpp:
`connect(this, &DebuggerForm::connected, viewer, &TileViewer::refresh);`